README.md

```diff
@@ -125,7 +125,7 @@ export default defineConfig({
 import { vi } from "vitest";
 
 vi.mock("./libs/client", () => ({
-  client: vPrisma.client,
+  prisma: vPrisma.client,
```

Comment:
@5t111111 thanks for noticing my mistake.

I think it is line 91 that needs to be corrected, not here.
Because the example in `libs/user.ts` (line 96) imports `client.

Reply:
@aiji42 ah, good point! going to push changes. Please let me know if you prefer squashing commits and force pushing.